test/test-container-start-command.js

```diff
@@ -10,7 +10,7 @@ function container() {
 }
 
 tap.test('start default', function(t) {
-  var s = new container();
+  var s = newContainer();
```

Comment:
It was the use of a non-capitalized constructor.. which in this case we aren't calling a constructor directly. Before this change it was actually a double `new`.

```js
function container() {
  return new Container();
}
var c = new container();
```